EIPS/eip-1822.md

```diff
@@ -150,21 +150,21 @@ Compatibility check to ensure the new Logic Contract implements the Universal Up
 
 ##### `updateCodeAddress`
 
-Stores the Logic Contract's address at storage `keccak256("PROXIABLE")` in the Proxy Contract.
+Stores the Logic Contract's address at storage slot `bytes32(uint256(keccak256('eip1967.proxy.implementation')) - 1)` in the Proxy Contract.
 
-The contract below shows the proposed implementation of the Proxiable Contract.
+The contract below shows a proposed implementation of the Proxiable Contract. In this example the compatibility check uses keccak256("PROXIABLE") for `proxiableUUID()`.
```

Comment:
Why are we leaving the old Proxiable stuff in here?

Reply:
Instead of using PROXIABLE or PROXIABLE-ERC1822-v1, should we change it everywhere to match the eip1967 style?

> eip1822.proxiable

or

> eip1822.proxiable.check